src/test/java/seedu/address/logic/parser/property/EditPropertyCommandParserTest.java

```diff
@@ -0,0 +1,232 @@
+        assertParseSuccess(parser, userInput, expectedCommand);
+
+        // price supplied only
+        userInput = targetIndex.getOneBased() + PRICE_DESC_PROPERTY1;
+        descriptor = new EditPropertyDescriptorBuilder()
+                .withPrice(VALID_PRICE_PROPERTY1).build();
+        expectedCommand = new EditPropertyCommand(targetIndex, descriptor);
+        assertParseSuccess(parser, userInput, expectedCommand);
+
+        // address supplied only
+        userInput = targetIndex.getOneBased() + ADDRESS_DESC_PROPERTY1;
+        descriptor = new EditPropertyDescriptorBuilder()
+                .withAddress(VALID_ADDRESS_PROPERTY1).build();
+        expectedCommand = new EditPropertyCommand(targetIndex, descriptor);
+        assertParseSuccess(parser, userInput, expectedCommand);
+
+        // description supplied only
+        userInput = targetIndex.getOneBased() + DESCRIPTION_DESC_PROPERTY1;
+        descriptor = new EditPropertyDescriptorBuilder()
+                .withDescription(VALID_DESCRIPTION_PROPERTY1).build();
+        expectedCommand = new EditPropertyCommand(targetIndex, descriptor);
+        assertParseSuccess(parser, userInput, expectedCommand);
+
+        // characteristics supplied only
+        userInput = targetIndex.getOneBased() + CHARACTERISTICS_DESC_PROPERTY1;
+        descriptor = new EditPropertyDescriptorBuilder()
+                .withCharacteristics(VALID_CHARACTERISTICS_PROPERTY1).build();
+        expectedCommand = new EditPropertyCommand(targetIndex, descriptor);
+        assertParseSuccess(parser, userInput, expectedCommand);
+
+        // owner name supplied only
+        userInput = targetIndex.getOneBased() + OWNER_NAME_DESC_PROPERTY1;
+        descriptor = new EditPropertyDescriptorBuilder()
+                .withOwnerName(VALID_OWNER_NAME_PROPERTY1).build();
+        expectedCommand = new EditPropertyCommand(targetIndex, descriptor);
+        assertParseSuccess(parser, userInput, expectedCommand);
+
+        // owner phone supplied only
+        userInput = targetIndex.getOneBased() + OWNER_PHONE_DESC_PROPERTY1;
+        descriptor = new EditPropertyDescriptorBuilder()
+                .withOwnerPhone(VALID_OWNER_PHONE_PROPERTY1).build();
+        expectedCommand = new EditPropertyCommand(targetIndex, descriptor);
+        assertParseSuccess(parser, userInput, expectedCommand);
+    }
+
+    @Test
+    public void parse_multipleValidRepeatedFields_acceptsLast() {
+        Index targetIndex = INDEX_FIRST_ITEM;
+        String userInput = targetIndex.getOneBased() + NAME_DESC_PROPERTY1 + NAME_DESC_HOME;
+
+        EditPropertyDescriptor descriptor = new EditPropertyDescriptorBuilder().withName(VALID_NAME_HOME).build();
+
+        EditPropertyCommand expectedCommand = new EditPropertyCommand(targetIndex, descriptor);
+        assertParseSuccess(parser, userInput, expectedCommand);
+    }
+
+    @Test
+    public void parse_invalidValueFollowedByValidValue_success() {
```

Comment:
This seems a bit weird to me, but I think it should be fine as long as we specified in the UG that we will only take the last parameter if you specify multiple fields.

Reply:
Sure! I'll take note to include it in the `notes about the command format` section of the guide.